regression/verilog/expressions/this1.desc

```diff
@@ -0,0 +1,7 @@
+CORE
+this1.sv
+
+^file .* line 3: 'this' outside of method$
```

Comment:
Shouldn't we also have a test where it is actually permitted?

Reply:
This would need an implementation of classes and methods.